src/components/Contacts/ContactDetails/ContactTasksTab/ContactTaskRow/ContactTaskRow.test.tsx

```diff
@@ -241,7 +241,7 @@ describe('ContactTaskRow', () => {
     it('displays Appointment', () => {
```

Comment:
This whole section could be a lot shorter if we used `it.each`.

Reply:
I have made the entire test shorter.